web/vue.config.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
   configureWebpack: {
     devtool: 'source-map',
   },
+  transpileDependencies: [
+    'vuetify',
+  ],
```

Comment:
I could not determine what this achieves. The project was working without this code, so I guess there's some edge case we might run in to with a different project configuration?

Reply:
https://github.com/vuetifyjs/vue-cli-plugins/pull/58

:man_shrugging: